includes/Exceptions/ConnectWCAPIException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\API\Exceptions;
+
+defined( 'ABSPATH' ) or exit;
+
+/**
+ * Class Connect_WC_API_Exception
+ * @package SkyVerge\WooCommerce\Facebook\API\Exceptions
+ */
+class Connect_WC_API_Exception extends \Exception {
```

Comment:
That doc comment is a great place to document the purpose / intention of the exception! Pretty clear from name but maybe could be clearer.